STRUCTURE.md

```diff
@@ -0,0 +1,105 @@
+# SugarLabs.org Information Architecture
+
+This document proposes a structure of the Sugar Labs homepage, [sugarlabs.org](http://www.sugarlabs.org)
+
+## Conversion funnel 
+
+A [conversion funnel](https://www.google.com/search?q=conversion+funnel&tbm=isch) plans the site visitor stage goals.
+
+1. Strangers: Discovery, Awareness, Attract, Brand
+    * Facebook, YouTube, Snapchat, Instagram, Twitter, Press
+    * Measure impressions/reach
+
+2. Visitors: Landing Page, Realization of Need, Interest/Desire, Engage leads, Familiarity
+    * Assert the unmet needs the "one key fan" users face, and how the products/services addresse their needs
+    * Survey their needs/expectations
+    * Calls To Action
+    * Measure active daily users
+
+3. Leads: Consideration, Try Outs, Action Flows
```

Comment:
One idea that interests me is using an email mini-course.  This could involve collecting emails on the home page.  Then, we guide people through getting set up and deriving value Sugar.  First, this would involve general emails about how sugar integrates with the classroom, and other topics to build a positive relationship and trust.  Then, it is eaiser to ask for a time investment to download and try SoaS.  They will also actually know what to do in SoaS.  After that, we could do more!

One bit of software that we could use is https://www.mautic.org/ .   But probably the most important part of this is making up the emails.

Anyway, I've just been listing to a lot of stuff on this topic.  I'd love to hear other opinions.